src/components/carousel/carousel.spec.js

```diff
@@ -680,7 +628,6 @@ describe('SlideStyle', () => {
     wrapper = mount(<Slide
       transitionName={ () => {} }
```

Comment:
```suggestion
```
I think `transitionName` is a classic prop

Reply:
It is. Well spotted. Thank you.